### DISCUSSION TITLE
My work

### DISCUSSION THREAD

**Discussion by crazyLillo** [tesivo/tesivo_s_VTMB_italian_translation_for_Unofficial_Patch#53] [Revisione]
Metterò sempre qui tutti i miei lavori, solo quando saranno riveduti e corretti al 100% 
( bhe! diciamo al 99,9%. non sono Dio, potrebbe sempre capitare una virgolina fuori posto, comunque io non ne ho viste...)
Quindi i file che trovate qui sono tutti revisonati è pronti per il gioco. (avrò perso mezzo grado di vista a revisionare tutto).

[Old Vamp1r3.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7882357/Old.Vamp1r3.zip)

1) - Nella cartella "downtown la", trovate il file "prince1.dlg", vi ho solo inserito la traduzione dell'intro, il resto ancora non lo tocco. (Revisionato)
Percorso : Unofficial patch\dlg\downtown la.

2) - Nella cartella "haven", trovate il file "female_sire_whisper2.lip - (corretta la frase, da "vorrei", in "voglio".
Percorso :  Unofficial patch\sound\cinematic\santa monica\haven.

3) - Nella cartella "nines", trovate i file "line907_col_e.lip - "line908_col_e.lip - "line909_col_e.lip". (tradotte per l'intro, non lo erano).
Percorso :  Unofficial patch\sound\character\dlg\main charaters\nines.

4) - Nella cartella "prince1", trovate tutti i file lip che servono per i sottotitoli dell'intro. (ritradotto e revisionato).
Percorso :  Unofficial patch\sound\character\downtown la\prince1.

5) - La cartella "signs". (  nella cartella italiana, troppe manipolazioni, ecco perchè la scritta "cliccare per continuare" non ci stava,
o  non si vedeva affatto. perchè erano stati alterati i valori del posizionamento della scritta.
Quindi ho lavorato sui file originali in inglese che aveva i valori giusti, ho solo dovuto modificarli un pelino 
per far entrare la scritta al punto giusto senza sbordare.
Ora contiene tutti i 104 popup del tutorial di cui 2 non erano tradotti. Sistemata scritta "cliccare per continuare", 
in tutti i file, anche in quelli in cui la "maschera" era spostata in alto e la scritta non si vedeva perchè la sue coordinate erano fuori campo.
Nota : nella cartella "signs" ci sono anche altri file che son stati tradotti perchè non lo erano. La cartella "signs"  adesso è completata al 100%.
Percorso :  Unofficial_Patch\vdata\signs.

6) - Nella cartella "TUTORIAL", vi trovate il file "jack_tutorial.dlg", ricontrollato da cima a fondo, 
vi erano rimaste delle "E" maiuscole accentate e qualche lettera fuori posto. Adesso è ok.
Percorso :  Unofficial_Patch\dlg\main characters.

7) - Nella cartella "tutorial security guard-dlg", trovate il file "tutorial security guard.dlg" 
Ritradotti i dialoghi con la guardia di sicurezza nel sotteraneo. 
Percorso : Unofficial_Patch\dlg\downtown la.

8) - Nella cartella "tutorial_security_guard", invece, trovate i relativi file lip. Tradotti (non lo erano) .
Percorso : Unofficial_Patch\sound\character\dlg\downtown la\tutorial_security_guard.

9) - Nella cartella "system" troverete il file "string.txt".
le voci  in basso nell'inventario sono state tutte sistemate (prima sbordavano fuori scomparendo dalla vista.
Purtroppo si è dovuto trovare un compromesso traduttorio dato che lo spazio riservato a quelle scritte è di max 12 caratteri, non uno di più.
Si potrebbe provare a modificare la maschera (il riquadro dell'inventario con i relativi piccoli riquadri sotto) ingrandendoli.
ma non lo so quali siano i file interessati e non so quanto lavoro ci sia bisogno di fare, non ne ho la piu pallida idea, ma ci si può provare,
se il mio "aggiustamento" non dovesse essere soddisfacente. 
Ma da quello che vedo penso che possa andare bene, visto che non c'è un modo facile per farlo,  poi dite voi.
Percorso : Unofficial_Patch\vdata\system

10) - Nella cartella "newcaster" ci sono tutti i file corretti ("newcaster"  tradotto come "annunciatore radio", corretto con "Giornalista" dato che è un TV.)
Percorso : Unofficial_Patch\sound\character\dlg\generic\newscaster.

11) - Nella cartella "male" trovate tradotti i file lip delle frasi dette del tipo ubriaco che cerca le chiavi della machina (la primissima bevuta di sangue nel tutorial.)
Scommetto che nessuno se ne era accorto, preso dalla voglia di azzannarlo...
non parla subito, se aspetti e gli gironzoli intorno mettendoti davanti a lui intralciando, a un certo punto comincia a parlare.
dice tipo "Yeaa hmm hmmm" (spesso mentre fuma), sta un po zitto e in seguito se gli stiamo davanti per un pò, dice anche "what's wrong?". e può ripeterlo più volte anche, 
dipende da quanto lo intralciamo.
ho individuato i file incriminati e c'è ne sono piu di uno con la stessa frase. Bisogna tradurli tutti, perchè il sistema preleva in modo random,
la stessa frase ma da file diversi. Quindi ho tradotto la frase " Yeaa hmm hmmm " con " Si hmm hmmm " .
invece la frase " what's wrong? ", lo tradotta in due modi diversi. 1) - " Che cosa c'e' che non va'? " e  2) - " Cosa c'e' che non va'? " (senza il "che").
A video, le mostra ramdom tutte e due (una alla volta ovviamente). Avevo fatto prove, non traducendole tutte e il risultato è stato che
a video vedevo una volta in italiano e una volta in inglese, stupendomi, ma  ho capito il sistema random di scelta delle frasi.
Quelle frasi sono dette anche da altri npc. Ho tradotto tutte quelle che servivano al caso specifico.
Percorso : Unofficial_Patch\sound\character\male.

12) - Nella cartella "young thug" trovate dei file lip, sono le frasi dette da i due tipi armati che dobbiamo far fuori, alla fine del tutorial.
Percorso : Unofficial_Patch\sound\character\male\young_thug.

13) - Nella cartella "jack_tutorial" vi sono i file lip per alcuni sottotitoli di jack visualizzati a schermo.
Percorso : Unofficial_Patch\sound\character\dlg\main characters\jack_tutorial

Tutti questi file sono pronto per il gioco!

saluti

**Comment by NarakuITA** [comment]
Ottimo Lavoro!

**Reply by crazyLillo** [reply]
Grazie mille!

Nella cartella "colori", vi sono file che vanno nel percorso Unofficial_Patch\vdata\signs.

[colori.zip](https://github.com/tesivo/VTMB_italian_translation_for_Unofficial_Patch/files/7898181/colori.zip)


per avere questa modifica per capire le risposte colorate.
quando si potrà, cercherò di modificare l'immagine del riquadro in basso con le scritte a colori.

intanto è così adesso :

![meglio](https://user-images.githubusercontent.com/96891196/150158237-00818f9d-0bc7-4f36-874b-e440ec87abb8.jpg)

**Comment by tesivo** [comment]
Complimenti per il lavoro, unirò le tue modifiche alla traduzione a breve.

**Reply by crazyLillo** [reply]
Si, ma questo si riferisce alla patch plus, che avevo installato per sbaglio

**Reply by tesivo** [reply]
Non ti preoccupare, prima o poi si tradurrà anche questo quindi ben venga anche questo lavoro ;)
